assets/airports/essa.json

```diff
@@ -0,0 +1,2865 @@
+{
+    "_comment": "Source airport info: https://www.aro.lfv.se/Editorial/View/7952/ES_AD_2_ESSA_en",
+    "airac": 2101,
+    "radio": {
+        "twr": "Arlanda Tower",
+        "app": "Stockholm Control",
+        "dep": "Stockholm Control"
+    },
+    "icao": "ESSA",
+    "iata": "ARN",
+    "magnetic_north": 6,
+    "ctr_radius": 150,
+    "ctr_ceiling": 9500,
+    "_comment": "Initial climb clearance is 3000 ft if not on a SID, all SIDs have 5000 ft",
+    "initial_alt": 3000,
+    "position": ["N59d39m07.00", "E017d55m07.00", "138ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": ["N59d39m07.00", "E017d55m07.00"],
+        "radius_nm": 5.0
+    },
+    "has_terrain": false,
+    "wind": {
+        "angle": 180,
+        "speed": 5
+    },
+    "arrivalRunway": "19L",
+    "departureRunway": "19R",
+    "defaultMaps": ["Arlanda CTR", "Stockholm TMA", "Sector Uppsala + Uppsala CTR", "Sector Västerås + Västerås CTR", "Bromma CTR"],
+    "_comment": "Airspaces below are simplifications of Stockholm TMA and Arlanda CTR where adjacent sectors are combined where possible. All real sectors are drawn in defaultMaps for realistic looks.",
+    "airspace": [
+        {
+            "_comment": "Arlanda CTR West + Arlanda CTR East + Overlapping part of Stockholm TMA b",
+            "floor":          0,
+            "ceiling":        9500,
+            "airspace_class": "C",
```

Comment:
This is taken directly from the reference. Does these docs and the FAA use the same definitions?

Reply:
See section 2.17 [here](https://www.aro.lfv.se/Editorial/View/8679/ES_AD_2_ESSA_en)

Reply:
I changed this to B inspired by the ENGM airport guide.